polyply/src/gen_coords.py

```diff
@@ -116,6 +116,7 @@ def gen_coords(toppath,
                max_force=5*10**4.0,
                nrewind=5,
                lib=None,
+               protocol='default',
```

Comment:
I think protocol is not a very good variable name here, because it is not directly obvious that it refers to back mapping. But I cannot think of a better yet short one.

Reply:
Agree, for now I'll go with '-bm_mode' and the variable 'bmode'